doc/holochain_101/src/nodejs_dna_instances.md

```diff
@@ -0,0 +1,61 @@
+# DNA Instances
+
+`DnaInstance` is a class that is exported from `holochain-nodejs` and can be imported into your code.
```

Comment:
```suggestion
`DnaInstance` is a class that is exported from `holochain-nodejs` and can be imported into your code.
This class is used externally and instances of it are built automatically for you to use, so you typically should not have to construct a `DnaInstance` yourself.
```

I know you wrote this in the notes for the constructor but seems good to have a note right up top too.